src/style.css

```diff
@@ -16,6 +16,18 @@ body {
   background: #f5f5f5;
 }
 
+.hidden {
+  visibility: hidden;
+  position: fixed;
+
+  /* top: 0; */
```

Comment:
[OPTIONAL]
- [ ] Please don't push commented out code to your code base, it makes your project looks messy. Kindly remove it

Reply:
I missed that particular line, thank you for pointing it out!